packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## NEXT
```

Comment:
This should be a new minor version bump 2.7.0

Reply:
this will now become 2.8.0 right?